contracts/ZkLink.sol

```diff
@@ -658,6 +632,36 @@ contract ZkLink is ReentrancyGuard, Storage, Events, UpgradeableMaster {
         withdrawOrStore(op.tokenId, rt.tokenAddress, rt.standard, rt.decimals, op.owner, op.amount);
     }
 
+    /// @dev Execute fast withdraw or normal withdraw according by nonce
+    function executeFastWithdraw(uint32 accountId, uint32 nonceFromAccountId, uint8 nonceFromSubAccountId, uint32 nonce, address owner, uint16 tokenId, uint128 amount, uint16 fastWithdrawFeeRate) internal {
```

Comment:
This function does normal withdraw and fast withdraw, but name is `executeFastWithdraw`. I think `_executeWithdraw` is better.